Sources/Clickstream/Dependency/Clickstream/AutoRecord/AutoRecordEventClient.swift

```diff
@@ -170,7 +170,7 @@ class AutoRecordEventClient {
     }
 
     func recordSessionStartEvent() {
-        let event = clickstream.analyticsClient.createEvent(withEventType: Event.PresetEvent.SESSION_START)
+        let event = clickstream.analyticsClient.createEvent(withEventType: Event.PresetEvent.SESSION_START)!
```

Comment:
Why creating event returning null?

Reply:
because when the event name is invalid this method will return nil, this does seem inappropriate. The next commit will separate the event check from event record method and fix it.